portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +53,24 @@ function rotateImages() {
   
   setTimeout("rotateImages()", time);
 }
+
+function getCommentsUsingArrows() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    commentsList = document.getElementById('comment-container');
+    commentsList.innterHTML = '';
+    commentsList.appendChild(
+      createListElement(comments[0]));
```

Comment:
It's probably best to use a loop here rather than hard-code index values:

```
comments.forEach((comment) => {
  commentsList.appendChild(createListElement(comment));
}
```